drivers/gpio/gpio_mcux_lpc.c

```diff
@@ -429,8 +427,7 @@ static const clock_ip_name_t gpio_clock_names[] = GPIO_CLOCKS;
 		.gpio_base = GPIO,							\
```

Comment:
@VitekST Lets put this in another PR so that we can merge it faster. 

As per discussion with @stephanosio  the toolchain will be available at the beginning of February.

https://github.com/zephyrproject-rtos/sdk-ng/pull/721#issuecomment-1886648261

Reply:
@dbaluta Okay, I'll create one.

Reply:
@dbaluta Here you go: https://github.com/zephyrproject-rtos/zephyr/pull/67667